test.py

```diff
@@ -22,5 +22,5 @@ def bubbleSort(array):
 
 bubbleSort(data)
 
-# print('Sorted Array in Ascending Order:')
+print('Sorted Array in Ascending Order:')
```

Comment:
[test.py, line 25]: The comment on line 25 is not necessary and serves no purpose.